libs/npm/core/src/components/content/DroppableContentContainer.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
will this replace the one that Jonny implemented for all drag and drop content?

Reply:
The goal is to have a shared component that is reused to eliminate the amount of work it is to add the functionality everywhere it is needed.  We'll switch over to using the new component slowly.